apps/website/src/examples/Input.main.tsx

```diff
@@ -3,12 +3,14 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { Input } from '@itwin/itwinui-react';
+import { Input, LabeledInput } from '@itwin/itwinui-react';
 
 export default () => {
   return (
-    <div>
+    <div style={{ display: 'flex', gap: '8px', flexDirection: 'column' }}>
       <Input placeholder='Basic input' />
+      <LabeledInput label='Labeled input' message='Input message' />
```

Comment:
I think we should only show one example, and only of `LabeledInput`. Other examples can be shown on the in the rest of the page.

Reply:
Showing only one example in main example.